CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ### Unreleased
   - Add all service specific config to config file
+  - Change to use pytest to improve test output and code coverage stats
```

Comment:
PyTest doesn't improve our coverage stats

Reply:
missing and will change